release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -3037,3 +3037,54 @@ link:https://access.redhat.com/solutions/6681391[{product-title} 4.8.29 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-8-31"]
+=== RHBA-2022:0484 - {product-title} 4.8.31 bug fix and security update
+
+Issued: 2022-02-15
+
+{product-title} release 4.8.31 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:0484[RHBA-2022:0484] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:0483[RHSA-2022:0483] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
```

Comment:
For the second sentence, it is generally advisable to put the prefacing phrase first, and then the xref: 

"For notes on the container images in this release, see the following article:"